sync.py

```diff
@@ -72,12 +82,33 @@ def sync_directly(unsycned_builds):
         print("Removing " + build['nvr'] + ".src.rpm...")
         os.remove(build['nvr'] + ".src.rpm")
 
+local_packages = False
 # Badly written but working python script
 if __name__ == "__main__":
     # tag = sys.argv[1]
     tag = "rhel-8.2.0-candidate"
+    ctag = "dist-c8-stream-compose"
+
     brew_proxy = brew.ClientSession("http://brewhub.engineering.redhat.com/brewhub/")
-    packages_to_track = load_package_list()
+    if local_packages:
+        packages_to_track = load_package_list()
+    else:
+        print("Contacting mbox for:", ctag)
+        cproxy = brew.ClientSession("https://koji.mbox.centos.org/kojihub/")
+        ctagged_builds = get_tagged_builds(cproxy, ctag)
+        packages_to_track = [b['package_name'] for b in ctagged_builds]
+        # Now filter out the pacakges which are already uptodate in CentOS
+        latest = set()
+        for build in ctagged_builds:
+            latest.add(build['nvr'])
+        ntagged = []
+        for build in sorted(tagged_builds, key=lambda x: x['package_name']):
```

Comment:
Shall this be `ctagged_builds` ?

Reply:
No, this is filtering out the packages from rhel which are already in CentOS.

Reply:
This is the output on my devel vm:
```
$ python3 sync.py 
/usr/local/lib/python3.6/site-packages/rpm.py:15: UserWarning: The RPM Python bindings are not currently available via PyPI.

Please install them with your distro package manager (typically called
'python2-rpm' or 'python3-rpm'), and ensure that any virtual environments
needing the API are configured to be able to see the system site packages
directory.

  warnings.warn(warning_msg)
Contacting mbox for: dist-c8-stream-compose
Traceback (most recent call last):
  File "sync.py", line 105, in <module>
    for build in sorted(tagged_builds, key=lambda x: x['package_name']):
NameError: name 'tagged_builds' is not defined
```

Reply:
Should be fixed now, missed moving a line in the merge.